fastapi_template/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/db_sa/models/users.py

```diff
@@ -0,0 +1,84 @@
+import uuid
+from typing import Optional
+
+from fastapi import Depends, Request
+from fastapi_users import BaseUserManager, FastAPIUsers, UUIDIDMixin, schemas
+from fastapi_users.authentication import (AuthenticationBackend, BearerTransport,
+                                          CookieTransport, JWTStrategy)
+from fastapi_users.db import SQLAlchemyBaseUserTableUUID, SQLAlchemyUserDatabase
+from httpx_oauth.clients.google import GoogleOAuth2
+from sqlalchemy.ext.asyncio import AsyncSession
+from {{cookiecutter.project_name}}.db.base import Base
+from {{cookiecutter.project_name}}.db.dependencies import get_db_session
+from {{cookiecutter.project_name}}.settings import settings
+
+SECRET = settings.users_secret
+
+
+class User(SQLAlchemyBaseUserTableUUID, Base):
+    pass
+
+
+class UserRead(schemas.BaseUser[uuid.UUID]):
+    pass
+
+
+class UserCreate(schemas.BaseUserCreate):
```

Comment:
Also lack of documentation will fail flake8 linter on pre-commit. Please add docs with pep8 format for all classes and functions.